content/sensu-go/5.13/reference/assets.md

```diff
@@ -690,3 +870,11 @@ It is also worth noting that deleting an asset does not delete the archive or do
 [31]: #example-asset-with-a-check
 [32]: ../../installation/plugins/#installing-plugins-using-the-sensu-install-tool
 [33]: ../../installation/configuration-management/
+
+[asset-ref]: https://docs.sensu.io/sensu-go/latest/reference/assets/
+[gh-release-01]: http://share.sachshaus.net/4efc554512f9/%255Bee92b1343de6399b8191fee8b8dd2c57%255D_Image%2525202019-09-17%252520at%25252010.32.02%252520AM.png
```

Comment:
Please copy these images into the docs somewhere under `static/images/` so we can avoid serving them from your personal site.